autopush_rs/src/client.rs

```diff
@@ -479,7 +524,7 @@ where
                     Either::A(ClientMessage::Nack { .. }) => {
                         self.data.srv.metrics.incr("ua.command.nack").ok();
                         self.data.webpush.as_mut().unwrap().stats.nacks += 1;
-                        ClientState::WaitingForAcks
+                        ClientState::Await
```

Comment:
This was a  bug I noticed and fixed here, as its supposed to return to the prior state.